man/systemd.network.xml

```diff
@@ -1340,6 +1340,15 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             to <literal>ipv4</literal>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>OnLink=</varname></term>
+          <listitem>
+            <para>Takes a boolean. If set to true, the kernel does not have to check if the gateway is
+            reachable directly by the current machine (i.e., the kernel does not need to check if the
+            gateway is attached to the local network), so that we can insert the nexthop in the kernel
+            table without it being complained about. Defaults to <literal>no</literal>.</para>
```

Comment:
It's also enabled automatically in one case, no? So maybe "Defaults to no, unless Gateway= is set without a static address configured, in which case it is enabled"

Reply:
I'd like to change the condition in the future. So, I'd like to not mention that. If no address is configured, then we emit a warning. So, I hope it is enough now.

Reply:
Ok, makes sense